tests/data_context/test_data_context.py

```diff
@@ -390,6 +390,25 @@ def test_data_context_get_validation_result(titanic_data_context):
     assert len(failed_validation_result.results) == 8
 
 
+def test_data_context_get_latest_validation_result(titanic_data_context):
+    """
+    Test that the latest validation result can be correctly fetched from the configured results
+    store
+    """
+    for _ in range(2):
+        titanic_data_context.profile_datasource("mydatasource")
+    assert len(titanic_data_context.validations_store.list_keys()) > 1
```

Comment:
Could we change this to 
```suggestion
    assert len(titanic_data_context.validations_store.list_keys()) == 2
```
to be more explicit?